autode/log/methods.py

```diff
@@ -0,0 +1,47 @@
+
+
+class ComputationalMethods:
+    """Generic computational methods section built by autodE"""
+
+    def __add__(self, other):
+        """Add another sentence to the methods if it's not already present"""
+        assert type(other) is str
+
+        if other not in self._list:
+            self._list.append(other)
+
+        return None
+
+    def __repr__(self):
+        return self.__str__()
+
+    def __str__(self):
+        """String of the computational methods used in this initialisation
+        of autodE prepended with the current version"""
+        import autode
+        autode_str = (f'All calculations were performed in autodE '
+                      f'v. {autode.__version__}. ')
+
+        return autode_str + " ".join(self._list)
+
+    def add(self, other):
```

Comment:
ew

Reply:
what's wrong with this?!

Reply:
same as the string thing, this function only exists to call another function, seems unnecessary

Reply:
I know what you mean but for a class it seems cleaner to do methods.add('thing') rather than methods += 'thing', but both should be possible